tsdb/index/tsi1/cache_test.go

```diff
@@ -15,7 +15,7 @@ func TestTagValueSeriesIDCache(t *testing.T) {
 	m0k1v2 := tsdb.NewSeriesIDSet()
 	m1k3v0 := tsdb.NewSeriesIDSet(900, 0, 929)
 
-	cache := TestCache{NewTagValueSeriesIDCache(10)}
+	cache := TestCache{NewTagValueSeriesIDCache(10, nil)}
```

Comment:
I'd be interested in trying to replace these `nil` values with actual trackers, and extending the tests to assert on the collected values.